bin/setup

```diff
@@ -59,9 +61,15 @@ Dir.chdir APP_ROOT do
   run "gem install mailcatcher"
 
   puts "\n== Preparing database =="
-  run 'bin/rake db:reset RAILS_ENV=development'
+  run 'dropdb upaya_development'
+  run 'bin/rake db:create RAILS_ENV=development'
+  run 'bin/rake db:migrate RAILS_ENV=development'
+  run 'bin/rake db:seed RAILS_ENV=development'
+  run 'dropdb upaya_development'
+  run 'bin/rake db:create RAILS_ENV=test'
+  run 'bin/rake db:migrate RAILS_ENV=test'
+  run 'bin/rake db:seed RAILS_ENV=test'
```

Comment:
What about this? This works for me without needing to run migrations:
```ruby
run 'bin/rake db:create'
run 'bin/rake db:create RAILS_ENV=test'
run 'bin/rails db:environment:set RAILS_ENV=development'
run 'bin/rake db:reset RAILS_ENV=development'
run 'bin/rails db:environment:set RAILS_ENV=development'
run 'bin/rake dev:prime RAILS_ENV=development'
run 'bin/rails db:environment:set RAILS_ENV=test'
run 'bin/rake db:reset RAILS_ENV=test'
```
and for Docker:
```ruby
run 'docker-compose run web rake db:create'
run 'docker-compose run web rake db:create RAILS_ENV=test'
run 'docker-compose run web bin/rails db:environment:set RAILS_ENV=development'
run 'docker-compose run web bin/rails db:reset RAILS_ENV=development'
run 'docker-compose run web bin/rails db:environment:set RAILS_ENV=development'
run 'docker-compose run web rake dev:prime'
run 'docker-compose run web bin/rails db:environment:set RAILS_ENV=test'
run 'docker-compose run web bin/rails db:reset RAILS_ENV=test'
```

Reply:
Taking inspiration from your thoughts, I updated to use `db:environment:set` and it seems to work, still feels weird. We don't need the initial `db:create` since `db:reset` handles that.